test/jdk/java/util/Calendar/bug4243802.java

```diff
@@ -24,70 +24,71 @@
 /*
  * @test
  * @bug 4243802
- * @summary confirm that Calendar.setTimeInMillis() and
- * getTimeInMillis() can be called from a user program. (They used to
- * be protected methods.)
+ * @summary confirm that Calendar.setTimeInMillis() and getTimeInMillis()
+ *          can be called from a user program. They used to be protected methods.
  * @library /java/text/testlib
+ * @run junit bug4243802
  */
 
-import java.util.*;
+import java.util.Calendar;
+import java.util.Locale;
+import java.util.TimeZone;
 
-public class bug4243802 extends IntlTest {
+import org.junit.jupiter.api.BeforeEach;
 
-    public static void main(String[] args) throws Exception {
-        new bug4243802().run(args);
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class bug4243802 {
+
+    @BeforeEach
+    void setUp() {
+        Locale.setDefault(Locale.US);
+        TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"));
```

Comment:
If the test is not going to restore the original defaults with `tearDown()`, I'd expect `othervm` explicitly on `@run` directive. (applies to other locations)

Reply:
Thanks for correcting that, I had a misconception that JUnit had a life cycle that had a new VM per test (but in reality it is a new test class instance life cycle), which is why I removed the saving of the default TZ and Locale. Adjusted so that they are preserved and reset with a tearDown() method.